lib/tasks/import_users.rake

```diff
@@ -52,4 +52,35 @@ namespace :import_users do
       user.update(update)
     end
   end
+
+  task csv: :environment do
+    file, client = ENV['FILE'], ENV['CLIENT']
+    if !file
+      raise 'FILE must be specified. e.g. rake import_users:csv FILE=/path/to.csv CLIENT=gsa18f'
+    elsif !client
+      raise 'CLIENT must be specified. e.g. rake import_users:csv FILE=/path/to.csv CLIENT=gsa18f'
+    end
+
+    data = CSV.read(ENV['FILE'], headers: true)
+    headers = data.headers()
+    first_col = ENV['FIRST_NAME_COL'] || headers.find {|header| header.downcase.include? "first"}
+    last_col = ENV['LAST_NAME_COL'] || headers.find {|header| header.downcase.include? "last"}
+    email_col = ENV['EMAIL_COL'] || headers.find {|header| header.downcase.include? "email"}
+    if !first_col || !last_col || !email_col
+      raise "Couldn't determine one or more of first name, last name, email: #{headers}"
+    end
+
+    data.each do |row|
+      email = row[email_col]
+      if !email
+        warn "Email is empty: " + row.inspect
+      else
+        User.for_email(email.strip).update(
```

Comment:
Makes sense -- I think there might even be an existing library somewhere in our app